tests/stub/configuration_hints/scripts/2_seconds_in_time_tx.script

```diff
@@ -45,7 +45,4 @@ S: <SLEEP> 1
        <SLEEP> 1
        SUCCESS {}
 *}
-{?
-    C: GOODBYE
-    S: <EXIT>
-?}
```

Comment:
Is the `?: RESET` not needed anymore?

Reply:
What `?: RESET` ?.? There never was one or am I not seeing it?

Reply:
It was my tired mind, i will resolve this conversation.